test/src/main/java/org/openjdk/skara/test/TestPullRequest.java

```diff
@@ -254,9 +255,14 @@ public void updateCheck(Check updated) {
 
     @Override
     public void setState(State state) {
+        data.state = state;
         data.lastUpdate = ZonedDateTime.now();
     }
 
+    public boolean isOpen() {
```

Comment:
```suggestion
    boolean isOpen() {
```
Can perhaps keep this one package private.